### DISCUSSION TITLE
Solicitação de domínio da Python Nordeste 2025 para o site

### DISCUSSION THREAD

**Discussion by anapaulamendes** [apyb/comunidade#150] [Geral]
Oi, pessoal!

Gostaríamos de solicitar o domínio http://2025.pythonnordeste.org/ para a Python Nordeste 2025. Vamos começar o trabalho de criação do site.

cc: @jjpaulo2

**Comment by cecivieira** [comment]
Eu posso estar errada, mas acho que é só tu configurar o domínio no CNAME e isso vai funcionar.

@naanadr @itepifanio tu lembra como foi nos anos anteriores?

**Reply by naanadr** [reply]
@anapaulamendes após configurar o CNAME (exemplo o [repo de 2024](https://github.com/pythonNordeste/pyne2024/blob/main/CNAME)) você volta aqui na thread para que eu possa atualizar o Cloudflare com o subdomínio 2025.

**Reply by jjpaulo2** [reply]
@naanadr consegue dar essa força?

![image](https://github.com/user-attachments/assets/bb2309eb-303c-48a3-97f1-891ec2ea4f6b)
